abdiff/core/utils.py

```diff
@@ -53,6 +53,14 @@ def update_or_create_run_json(run_directory: str, new_data: dict) -> dict:
 def create_subdirectories(
     base_directory: str, subdirectories: list[str]
 ) -> tuple[str, ...]:
+    """Create subdirectories nested within a base directory.
+
+    This util is preferred for commands that require creating
+    nested subdirectories to organize outputs (e.g., run_ab_transforms).
+
+    Returns:
+        tuple[str, ...]: A tuple of absolute paths to created subdirectories.
+    """
     directories = []
     for subdirectory in subdirectories:
         directory = Path(base_directory) / subdirectory
```

Comment:
Same with this one, it gets converted to a `str` anyway on line 68

Reply:
See https://github.com/MITLibraries/transmogrifier-ab-diff/pull/6#discussion_r1765349539.